Example/URBNConvenience/URBNTableViewHelper.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  URBNTableViewHelper.swift
```

Comment:
same here.. and maybe like TableViewControllerHelper? thoughts? Probably want to avoid having both `TableViewHelper` and `TableViewHelpers`